Scripts/SwiftLint.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
-SWIFTLINT="$(PWD)/Pods/SwiftLint/swiftlint"
-CONFIG="$(PWD)/.swiftlint.yml"
+PROJECT_DIR="${PROJECT_DIR:-`cd "$(dirname $0)/..";pwd`}"
```

Comment:
Note: I missed your last comment about `trap` & stuff 😅

Reply:
It not necessary if you do like you did, because using backticks like you did here spawns a new, independent shell to run the `cd "$(dirname $0)/..";pwd` whole command at once in a separate shell and returns the result. So because that full command is in a separate shell it shouldn't affect the current working directory of the current shell (well… I _think_).